doc/tutorials/plugins.md

```diff
@@ -419,8 +421,9 @@ keyNew ("system/elektra/modules/" ELEKTRA_PLUGIN_NAME "/exports/checkconf", KEY_
 ```
 
 Within the `checkconf` function all of the plugin configuration values should be validated.
-Errors should be reported via Elektra's error handling mechanism (see section [ELEKTRA_SET_ERROR](#elektra_set_error) for further details).
-If `checkconf` encounters a configuration value, that is not strictly invalid but can not be parsed by the plugin (e.g. a parameter which is not part of the plugin configuration), then a warning should be appended to `errorKey`, using `ELEKTRA_ADD_WARNING`.
+Errors should be reported via Elektra's error handling mechanism (see section [ELEKTRA*SET*<CONCRETE>\_ERROR](#elektra_set_concrete_error) for further details).
+If `checkconf` encounters a configuration value, that is not strictly invalid but can not be parsed by the plugin (e.g. a parameter which is not part of the plugin configuration),
+then a warning should be appended to `errorKey`, using `ELEKTRA_ADD_<CONCRETE>_WARNING`.
 
```

Comment:
Not mentioned that there is also `ERRORF`?

Reply:
It was not documented before but I may add it

Reply:
What do you mean with "may"?

Reply:
I "can" add it is more appropriate. I added the text already

Reply:
Thank you!

Reply:
It does not really explain how to use ERRORF or WARNINGF. Furthermore, at some places there is still `<CONCRETE>` used. I merge it nevertheless, as it is a small improvement.